baybe/simulation/scenarios.py

```diff
@@ -136,7 +137,13 @@ def unpack_simulation_results(array: DataArray) -> pd.DataFrame:
     # Simulate and unpack
     result_variable = "simulation_result"
     batch_simulator = make_xyzpy_callable(result_variable)
-    da_results = batch_simulator.run_combos(combos)[result_variable]
+
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
```

Comment:
Wouldn't it make sense to restrict to the specific warning type we want to filter out here?

Reply:
I already specify the particular module. (which currently doesnt throw any other warning types), is that not enough?

Reply:
I think you've answered the question yourself: the emphasis is on `currently`. Why not just pass the exact warning type via `category`?

Reply:
Would say in addition. Let's keep as narrow as possible for now

Reply:
ok, added

Reply:
Thanks! I think a comment would be helpful for future us to understand which situations cause the condition